Sources/Logging/Logging.swift

```diff
@@ -527,17 +527,16 @@ internal struct StdioOutputStream: TextOutputStream {
 
 // Prevent name clashes
 #if os(macOS) || os(tvOS) || os(iOS) || os(watchOS)
-let systemStderr = Darwin.stderr
-let systemStdout = Darwin.stdout
+    let systemStderr = Darwin.stderr
+    let systemStdout = Darwin.stdout
 #else
-let systemStderr = Glibc.stderr!
-let systemStdout = Glibc.stdout!
+    let systemStderr = Glibc.stderr!
+    let systemStdout = Glibc.stdout!
 #endif
```

Comment:
nit / question: I was under the impression "official formatting" for #if is to not indent? 

Either way, happy with having an automatted formatted so whichever we want is fine, just confused about this one.

Reply:
personally don't have a strong opinion on concrete format rules. what we agreed on in the past is capture in the repo's `.swiftformat` file, so we can change it if we want to agree on a different set of rules

Reply:
Yeah same, though this one is surprising... curious if choice was conscious on the `#ifs` @weissi ? As mentioned, not an issue and LGTM as I'm more than happy to have automated formatter

Reply:
I'd go with whatever Xcode does just because it's a pain otherwise because it'll re-indent in the way it thinks it's right.

Reply:
https://github.com/apple/swift-log/pull/84/commits/967999c856cc497c0d6831e1b99db01fd17a03ce